spec/request/post911_gi_bill_status_request_spec.rb

```diff
@@ -27,40 +27,31 @@
   # TODO(AJD): this use case happens, 500 status but unauthorized message
```

Comment:
I changed expectations in this spec based on https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/EVSS%20Integration/Letters%20and%20GIBS%20error%20messages%20mapping.md